### PR TITLE
Jormungandr: Answer when krakens are dead to /v1/journeys with ids

### PR DESCRIPTION
**Pull request by l-vincent-l**

### CODE REVIEW

File: source/jormungandr/jormungandr/instance.py [hunk 1]
```diff
@@ -231,7 +234,21 @@ def has_id(self, id_):
...
+        except DeadSocketException:
+            return False
+
+
+    def has_coord(self, lon, lat):
+        return self.has_point(geometry.Point(lon, lat))
+
+    def has_point(self, p):
+        if self.geom:
+            logging.info("'{} WKT {}".format(self.name, json.dumps(geometry.mapping(self.geom))))
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/instance.py]: info?

File: source/jormungandr/jormungandr/instance_manager.py [hunk 2]
```diff
@@ -220,11 +211,10 @@ def _all_keys_of_id(self, object_id):
 
     def _all_keys_of_coord(self, lon, lat):
         p = geometry.Point(lon, lat)
-        instances = []
-        # a valid instance is an instance containing the coord and accessible by the user
-        for key, instance in self.instances.iteritems():
-            if instance.geom and instance.geom.contains(p):
-                instances.append(instance.name)
+        instances = [i.name for i in self.instances.itervalues() if i.has_point(p)]
+        logging.info("all_keys_of_coord(self, {}, {}) returns {}".format(lon, lat, instances))
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/instance_manager.py]: maybe debug too no ?

File: source/jormungandr/tests/routing_tests.py [hunk 3]
```diff
@@ -343,3 +343,20 @@ def test_shape_in_geojson(self):
...
+    """
+
+    def test_one_dead_region(self):
+        self.krakens_pool["basic_routing_test"].kill()
+
+        response = self.query("v1/journeys?from=stop_point:stopA&to=stop_point:stopB&datetime=20120614T080000",
+                              display=False)
+        eq_(len(response['journeys']), 2)
+        eq_(len(response['journeys'][0]['sections']), 1)
+        eq_(response['journeys'][0]['sections'][0]['type'], 'public_transport')
```
Comment (antoine-de) [inline source/jormungandr/tests/routing_tests.py:361]: could you add a test with debug param to check the region queried ?

**Comment by antoine-de** [comment]
Jenkins, test this please

**Pull request merged**